odxtools/cli/list.py

```diff
@@ -61,30 +61,26 @@ def print_summary(odxdb: Database,
 
         for proto in dl.protocols:
             if (can_rx_id := dl.get_can_receive_id(proto.short_name)) is not None:
-                rich.print(
-                    f"  [blue]CAN receive ID[/blue] for protocol '{proto.short_name}': 0x{can_rx_id:x}"
-                )
+                rich.print(f"  CAN receive ID for protocol '{proto.short_name}': 0x{can_rx_id:x}")
 
             if (can_tx_id := dl.get_can_send_id(proto.short_name)) is not None:
-                rich.print(
-                    f"  [blue]CAN send ID[/blue] for protocol '{proto.short_name}': 0x{can_tx_id:x}"
-                )
+                rich.print(f"  CAN send ID for protocol '{proto.short_name}': 0x{can_tx_id:x}")
 
         if dl.description:
             desc = format_desc(dl.description, indent=2)
-            rich.print(f" [blue]Description[/blue]: " + desc)
```

Comment:
Sure about removing colors?

Reply:
This is basically a matter of taste, and for my mine the new output of list was a bit too much. (Note that there are still colors, because `rich.print` adds them automatically. this just deals them down a bit.)